### PR TITLE
bump intl version to 0.18.0

### PR DESCRIPTION
**Pull request by ArtyomSasin**
intl: ">=0.15.2 < 0.18.0"

### CODE REVIEW

**Comment by ArtyomSasin** [comment]
@googlebot I signed it!

**Review by Lucasmaia435 — APPROVED** [review]

**Comment by itsbalamurali** [comment]
@mit-mit @nshahan approve, please.

**Review by babulpatel1309 — APPROVED** [review]
**Urgent => Needs to be merged ASAP**

**Review by babulpatel1309 — APPROVED** [review]

**Review by rafaelcmm — APPROVED** [review]

**Comment by rafaelcmm** [comment]
This needs to be merged ASAP

File: charts_flutter/test/user_managed_state_test.dart [hunk 1]
```diff
@@ -89,7 +89,10 @@ class TestChartState extends State<TestChart> {
       defaultInteractions: false,
     );
 
-    return new GestureDetector(child: chart, onTap: handleOnTap);
+    return new Directionality(
+      textDirection: TextDirection.ltr,
+      child: new GestureDetector(child: chart, onTap: handleOnTap),
+    );
```
Comment (rafaelcmm) [inline charts_flutter/test/user_managed_state_test.dart:95]: You can safely omit `new`

```suggestion
    return Directionality(
      textDirection: TextDirection.ltr,
      child: GestureDetector(child: chart, onTap: handleOnTap),
    );
```

**Comment by cedricDevWP** [comment]
Up

**Review by Lucasmaia435 — APPROVED** [review]

**Review by Lucasmaia435 — APPROVED** [review]

**Review by babulpatel1309 — APPROVED** [review]

**Comment by cedricDevWP** [comment]
I'm still worried ... a new version will be available?

**Comment by tony123S** [comment]
please fix it asap :)

**Comment by tony123S** [comment]
@mit-mit  @nshahan

**Comment by kevlar700** [comment]
#566 
#582 
#587

**Review by Cliff2103 — APPROVED** [review]

**Comment by babulpatel1309** [comment]
@googlebot I've signed it

**Comment by hdavidmb** [comment]
Also waiting for this package to be updated asap.

**Review by hdavidmb — APPROVED** [review]

**Review by frankyvij — APPROVED** [review]
LGTM

**Comment by ArtyomSasin** [comment]
@nshahan approve, please

**Comment by connlaoi** [comment]
@lorrainekan @mit-mit @nshahan @cbbraun 

Can we please see some movement here, the community has already resolved this and just needs one of you to finalize.

Google managed packages should really be prioritized internally when major releases like 2.0.1 come out. Something to keep an eye on moving forward perhaps.

We would all greatly appreciate not having to use dependency-overrides. ❤️ Thanks for your time whomever comes to the rescue 🎉

**Comment by agilob** [comment]
Google doesn't accept external contributions on any of their published flutter addons. Each and every readme contains:

>This project is developed internally at Google and published for **external consumption, external contributions unfortunately cannot be taken at this time.**

**Review by rodrigonepomuceno — APPROVED** [review]

**Comment by Simpler1** [comment]
With no updates in over a year, I think this package has been abandoned by Google.  It may be time to start migrating to [syncfusion_flutter_charts](https://pub.dev/packages/syncfusion_flutter_charts).

**Comment by agilob** [comment]
@Simpler1  no.

>Disclaimer: This is a commercial package. To use this package, you need to have either Syncfusion Commercial License or Free Syncfusion Community license. For more details, please check the LICENSE file.

**Review by theVinesh — APPROVED** [review]

**Comment by joleaf** [comment]
Google, please review :(

**Review by kevlar700 — APPROVED** [review]

**Comment by cbbraun** [comment]
Sorry for the lack of response so far.

As you have picked up on, this specific project has been severely deprioritized and has only occasional volunteers trying to keep the lights on (and even there we haven't been able to keep up). While I am personally passionate about data visualizations on many platforms, the internal support we had evaporated. Flutter (and even Dart) isn't part of my normal development which makes it even harder to keep up on it.

In addition to the staffing/resource issue, the other inconvenient problem is that the "source" lives within google's repository and is pushed to github, which means it isn't a simple "approve pull request".

I'll see if I can get some of the smaller easy pull requests approved and pushed back out into github over the next week or so.

I hope to get the project revitalized in the future, even hoping to find a team to take the torch, but that'll take some internal backing that I've yet to see so far.

Thanks for you patience and sorry for the many inconveniences.

**Review by hey24sheep — APPROVED** [review]
lgtm

**Review by Netlob — APPROVED** [review]
lgtm. please merge asap

**Review by Pwuts — APPROVED** [review]
lgtm! taking way too long

**Comment by kevlar700** [comment]
Thank You @cbbraun

**Comment by hey24sheep** [comment]
Thank you @cbbraun  This fixes it but we still need version bump/update of charts

**Pull request closed without merging**